automatic/screencloud/tools/chocolateyInstall.ps1

```diff
@@ -0,0 +1,16 @@
+$ErrorActionPreference = 'Stop'
+
+$packageArgs = @{
+    packageName    = 'screencloud'
+    fileType       = 'msi'
+    softwareName   = 'ScreenCloud'
+
+    checksum       = '41b4a0c38dc677820add2a7b2a695e9f9e8e60a14e3e869abe5bfae27e54a2e5'
+    checksumType   = 'sha256'
+    url            = 'https://screencloud.net/files/windows/ScreenCloud-1.3.0-x86.msi'
+
+    silentArgs     = '/qn /norestart'
+    validExitCodes = @(0)
+}
```

Comment:
You're using incorrect properties for a embedded package.
You need to use the `file` property, and remove the `checkums*` and `url` properties.